app/models/enqueuer.rb

```diff
@@ -0,0 +1,34 @@
+class Enqueuer
+  def enqueue_faxes
+    SnapApplication.faxable.untouched_since(DELAY_THRESHOLD.minutes.ago).
+      find_each do |snap_application|
+      create_and_enqueue_export(destination: :fax,
+                                snap_application: snap_application)
+    end
+  end
+
+  def create_and_enqueue_export(params)
+    enqueue(Export.create(params))
+  end
+
+  def create_and_enqueue_export!(params)
+    enqueue(Export.create(params))
+  end
```

Comment:
What's the difference between the normal and bang methods? Is it necessary to have both since the method bodies are the same?

Reply:
! is supposed to `create!`, raising an exception in the event the export isn't valid.

Reply:
Should this be `enqueue(Export.create!(params))` (the `!` isn't there in the line above)

Reply:
Yep! Will fix Monday :)